### PR TITLE
ronin: 2.0.3 -> 2.0.4

### PR DESCRIPTION
**Pull request by Ch1keen**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/security/ronin/default.nix [hunk 1]
```diff
@@ -15,6 +15,14 @@ bundlerEnv {
     };
   };
 
+  postBuild = ''
+    for i in $out/bin/*; do
+      if [[ "$i" != *"/ronin"* ]]; then
```
Comment (bbenne10) [inline pkgs/tools/security/ronin/default.nix:20]: Isn't this just `find $out/bin -not -name "ronin*" -delete`? or am I missing something?

**Review by bbenne10 — COMMENTED** [review]
Other than the little bit in postBuild, this looks okay.

File: pkgs/tools/security/ronin/default.nix (hunk 1 above)
Comment (bbenne10) [reply pkgs/tools/security/ronin/default.nix:20]: Come to think of it - there's probably some bash extglob solution too, but I've never been particularly good at those...

Comment (Ch1keen) [reply pkgs/tools/security/ronin/default.nix:20]: No, you're right. I was not good at using `find` command and didn't know about extglob.

**Comment by Janik-Haag** [comment]
Result of `nixpkgs-review pr 245041` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ronin</li>
  </ul>
</details>

**Review by Janik-Haag — APPROVED** [review]
LGTM

**Review by bbenne10 — APPROVED** [review]
LGTM

**Comment by Ch1keen** [comment]
Thank you @bbenne10 @Janik-Haag

**Pull request merged**